pulp_rpm/app/migrations/0054_remove_gpg_fields.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 4.2.4 on 2023-09-08 14:45
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
Shouldn't we be preserving settings users may have made using the about-to-be-removed fields? If so - we should reorder the migrations to add `repo_config` field first, and then in this migration create a JSON file from the current settings and fill in that field as needed, and **then** delete the old fields.

wdyt?

Reply:
Same topic was brought here https://github.com/pulp/pulp_rpm/pull/3298#discussion_r1371733272

Reply:
I can still do it if you both asked about this. Don't have strong preference, but leaning towards what I did.

Reply:
I have no idea how I missed that whole discussion, brain skipped a groove.

The argument "we've disabled repo-file generation anyway" is compelling. I withdraw the proposal :)